packages/create-turbo/__tests__/cli.test.ts

```diff
@@ -63,30 +63,55 @@ describe("create-turbo cli", () => {
 
       switch (promptCount) {
         case 1:
-          expect(prompt).toEqual(
-            ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
-          );
+          try {
+            expect(prompt).toEqual(
+              ">>> Welcome to Turborepo! Let's get you set up with a new codebase."
+            );
+            done();
+          } catch (e) {
+            done(e);
+          }
+
           break;
         case 2:
-          expect(prompt).toEqual(
-            `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
-          );
-          cli.stdin.write(keys.enter);
+          try {
+            expect(prompt).toEqual(
+              `? Where would you like to create your turborepo? (./${DEFAULT_APP_NAME})`
+            );
+            cli.stdin.write(keys.enter);
+            done();
+          } catch (e) {
+            done(e);
+          }
+
           break;
 
         case 3:
           // Which package manager do you want to use?
           // easy to change deployment targets.
-          expect(getPromptChoices(prompt)).toEqual(["npm", "pnpm", "yarn"]);
+          const xx = getPromptChoices(prompt);
+          console.log(xx);
+          try {
+            expect(xx).toEqual(["npm", "pnpm", "- yarn (not installed)"]);
```

Comment:
We should be able to fix the `(not installed)` before landing this.

`yarn` is actually running, but it is being marked as not installed because `corepack` is erroring since the repository is configured for `pnpm`.